ui/app/AppMain.qml

```diff
@@ -15,6 +15,7 @@ import "./AppLayouts/Chat/popups"
 import "./AppLayouts/Chat/popups/community"
 import "./AppLayouts/Profile/Sections"
 import "./AppLayouts/stores"
+import "./AppLayouts/Profile/stores" as ProfileStores
```

Comment:
There is no need for an additional store instance, you can add this in the `status-desktop/ui/app/AppLayouts/stores/RootStore.qml` and assign it in line 462 like`store: appMain.rootStore.profileStore`.